src/html/escaped_string.cr

```diff
@@ -0,0 +1,34 @@
+class HTML::EscapedString
+  SUBSTITUTIONS = {
+    '&'  => "&amp;"
+    '\"' => "&quot;",
+    '\'' => "&apos;",
+    '<'  => "&lt;",
+    '>'  => "&gt;",
+  }
+
+  def self.escape(str)
+    new(str).to_s
+  end
+
+  def initialize(@str)
+    @escaped_str = StringIO.new
```

Comment:
I don't see how that's suposed to work. Unless I remove the constructor and use only class methods, which doesn't seem like the best approach in my opinion